lib/req_athena.ex

```diff
@@ -307,11 +307,27 @@ defmodule ReqAthena do
     {Request.halt(request), %{response | body: result}}
   end
 
-  defp decode_rows(rows, fields) do
-    Enum.map(rows, fn %{"Data" => columns} ->
-      Enum.with_index(columns, fn value, index ->
-        field = Enum.at(fields, index)
-        decode_value(value, field)
+  defp decode_column_labels(column_labels) do
+    column_labels
+    |> Enum.map(&Map.fetch!(&1, "VarCharValue"))
```

Comment:
```suggestion
   Enum.map(column_labels, &Map.fetch!(&1, "VarCharValue"))
```
(we usually use pipe if there are more than a single function call :))